### ISSUE TITLE
doc: query timeouts: cumulative or ?

### ISSUE THREAD

**Issue by gavanderhoorn** [marip8/sampling_kinematics_plugin#10]
The readme contains an example configuration blurb:

```yaml
ext_axis_manipulator:
  kinematics_solver: sampling_kinematics_plugin/ExternalAxisSamplingKinematicsPlugin
  kinematics_solver_search_resolution: 0.1
  kinematics_solver_timeout: 0.005
  kinematics_solver_attempts: 1
  robot_group: manipulator
  weights: [0.1, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0]
manipulator:
  kinematics_solver: ur_kinematics/UR5KinematicsPlugin
  kinematics_solver_search_resolution: 0.1
  kinematics_solver_timeout: 0.005
  kinematics_solver_attempts: 1
```

Of course, both groups need their own configuration, but it's unclear to me right now whether the `kinematics_solver_timeout` and `kinematics_solver_attempts` settings are cumulative or not (ie: when using the full group (rail + manipulator) does the timeout now become `0.01`)?

(I'm not really asking this for myself, more to discuss whether this should be clarified in the readme)

**Comment by gavanderhoorn** [comment]
Same for the `kinematics_solver_search_resolution`.

**Comment by marip8** [comment]
Closing; addressed in #12
